org.gaphor.Gaphor.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: org.gaphor.Gaphor
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+rename-icon: gaphor
+command: gaphor
+finish-args:
+- "--socket=x11"
```

Comment:
You can remove the quotes and indent according to the style guide:
https://github.com/flathub/flathub/wiki/YAML-Style-Guide

Reply:
Fixed, thanks for the correction!